grakn-engine/src/main/java/ai/grakn/engine/task/postprocessing/PostProcessingTask.java

```diff
@@ -81,23 +96,28 @@ public void run() {
      * @param keyspace The {@link Keyspace} requiring post processing for a specific index
      * @param index the index to be post processed
      */
-    public void processIndex(Keyspace keyspace, String index){
+    public void processIndex(Keyspace keyspace, String index, UUID executionId){
```

Comment:
how does this id help?

Reply:
@kasper-piskorski make logs more understandable from the user perspective